exec/java-exec/src/test/java/org/apache/drill/exec/compile/TestEvaluationVisitor.java

```diff
@@ -72,6 +73,37 @@ public void x() throws Exception{
     logger.debug(g.generateAndGet());
   }
 
+  @Test
+  public void testVisitBooleanOrWithoutFunctionsEvaluation() throws Exception {
```

Comment:
How about move the new testcase into TestBugFixes? The new testcases uses SQL, while the existing one in TestEvaluationVisitor actually tests in a lower level.  Probably better to not mix them together.